opcua/address_space.py

```diff
@@ -139,6 +139,7 @@ def _translate_browsepath_to_nodeid(self, path):
             res.StatusCode = ua.StatusCode(ua.StatusCodes.BadNodeIdInvalid)
             return res
         current = path.StartingNode
+        # FIXME: Should be more according to the Specs: Part 4 Page 44
```

Comment:
Just checked it. It is good. Yesterday seamed strange... sorry

Even though you are not checking inverse references and subtypes... I don't know if this is important.